src/lib/integration_test_cloud_engine/kubernetes_network.ml

```diff
@@ -22,60 +25,79 @@ module Node = struct
 
   let base_kube_args t = [ "--cluster"; t.cluster; "--namespace"; t.namespace ]
 
-  let run_in_postgresql_container node ~cmd =
-    let base_args = base_kube_args node in
-    let base_kube_cmd = "kubectl " ^ String.concat ~sep:" " base_args in
-    let kubectl_cmd =
-      Printf.sprintf "%s -c %s exec -i %s-0 -- %s" base_kube_cmd
-        node.container_id node.container_id cmd
+  (* let run_in_postgresql_container node ~cmd =
+     let base_args = base_kube_args node in
+     let base_kube_cmd = "kubectl " ^ String.concat ~sep:" " base_args in
+     let kubectl_cmd =
+       Printf.sprintf "%s -c %s exec -i %s -- %s" base_kube_cmd
+         node.container_id node.pod_id cmd
+     in
+     let%bind cwd = Unix.getcwd () in
+     Util.run_cmd_exn cwd "sh" [ "-c"; kubectl_cmd ] *)
+
+  let get_logs_in_container ?container node =
```

Comment:
Not sure: With only specifying `mina_container_id`, does this mean `dump_mina_logs` skips over the archive container id?

Reply:
in line 382, there's a dump_archive_data and a dump_mina_logs function, both of which call this get_logs_in_container function, but of course specifying different containers.  so yeah you only do one container at a time with this function